man/mytop.1

```diff
@@ -0,0 +1,7 @@
+.TH MYTOP "1" "March 2020" "MariaDB 10\&.5" "MariaDB Database System"
+.SH NAME
+mytop \- display MariaDB server performance info like 'top'
+.SH "SEE ALSO"
+For more information, please refer to the MariaDB Knowledge Base, available online at https://mariadb.com/kb/
```

Comment:
PR #215 did some corrections about `mytop`, but I don't know we have a proper documentation about `mytop` or is this work in progress based on MDEV-4476 @iangilfillan ?

Reply:
This man page is based on latest version of mytop, so it should be OK.